actstream/transformers.py

```diff
@@ -0,0 +1,5 @@
+class DefaultVerbTransformer(object):
+    verb_map = {}
```

Comment:
Please add a docstring to explain what this class is good for and ideally a small example/description on how to use it.